### DISCUSSION TITLE
20231118 rotary encoder setting not saved.

### DISCUSSION THREAD

**Discussion by noriadzik** [rern/rAudio#1504] [Bug Reports]
Setup: RPi 4 + DigiAMP+

When I'm trying to turn on rotary encoder, I get information, that it is enabled, but it's not working. Also, when I go to different tab/restart rAudio, "Rotary Encoder switch" is set to OFF

**Comment by rern** [comment]
Try: `pacman -Sy dtc` then disable > enable again.

**Reply by noriadzik** [reply]
Thanks rern, fix helped. Also I see, that on latest version (20231125) it is also working :)
